### PR TITLE
106 button state not relayed and aria mislabeled

### PR DESCRIPTION
**Pull request by csps-snap**
#105 #106

- removed aria
- changed text for button description
- redid the lock to be css-driven instead of inline svg
- added hover state to lock.

### CODE REVIEW

File: components/12-days/Flap.js [hunk 1]
```diff
@@ -26,9 +26,11 @@ function Flap(props) {
     }
 
     return (
-        <button className={styles.flap} data-day={props.day} onClick={open} aria-label={`${props.t.open} ${props.day}` }>
-            <p>{props.day}</p>
-            <IcomoonReact className={styles.lock} iconSet={iconSet} size={30} icon={checkDate() ? "lock-open" : "lock-locked"}/>
+        
+        <button className={styles.flap} data-day={props.day} onClick={open} >
+            <span class={styles.hiddenday}>Day </span>
```
Comment (romy-piche) [inline components/12-days/Flap.js:31]: I got a warning in React (you can see it in the browser console):

> Warning: Invalid DOM property `class`. Did you mean `className`?


Just need to replace `class ` with `className`

File: components/12-days/Flap.js [hunk 2]
```diff
@@ -26,9 +26,11 @@ function Flap(props) {
...
+            <span class={styles.day}>{props.day}</span>
+            <span class={ checkDate() ?styles.lockopen:styles.lockclosed }>{checkDate() ? "Unlocked" : "Locked"}</span>
```
Comment (romy-piche) [inline components/12-days/Flap.js:33]: I guess it is working without space, but just to be more readable, I prefer this way, what do you think ?

`checkDate() ? styles.lockopen : styles.lockclosed`

**Review by romy-piche — CHANGES_REQUESTED** [review]
Just see my comment for the `class ` warning, otherwise good job !

File: components/12-days/Flap.js (hunk 2 above)
Comment (csps-snap) [reply components/12-days/Flap.js:33]: great jvais réparer ça !

File: components/12-days/Flap.js (hunk 1 above)
Comment (csps-snap) [reply components/12-days/Flap.js:31]: cool! jrègle ça !

File: components/12-days/Flap.js (hunk 2 above)
Comment (romy-piche) [reply components/12-days/Flap.js]: Les traductions se retrouvent dans /locales ! Et pour utiliser le champ traduit, les traductions sont accessible avec la props t:
ex: `{checkDate() ? props.t.unlocked : props.t.locked}`

**Review by romy-piche — APPROVED** [review]
🥇 !

**Pull request merged**